unit_tests/fields/field_algo_base_test.cpp

```diff
@@ -534,11 +534,11 @@ string field_input = R"INPUT(
    init_conc=[ 10, 20, 30],    // FieldConst
    conductivity={ //3x3 tensor
```

Comment:
Revisit. Should be tensor here. Does the test pass?

Reply:
Input string is used in init_from_input test without assemblation. Test passes with both (vector and tensor) formula. But I changed input formula to tensor according to the comment.